### ISSUE TITLE
Use of eval is strongly discouraged as it poses security risks

### ISSUE THREAD

**Issue by andyslack** [CdTgr/vue3-q-tel-input#24]
Hi all,

The component is currently throwing up this error:

`ERROR Use of eval in "node_modules/vue3-q-tel-input/dist/vue3-q-tel-input.esm.js" is strongly discouraged as it poses security risks and may cause issues with minification.
`

Implementation is:

```
import Vue3QTelInput from 'vue3-q-tel-input'
import 'vue3-q-tel-input/dist/vue3-q-tel-input.esm.css'
```

Any suggestions?

**Comment by CdTgr** [comment]
@andyslack The actual error comes from the [google-libphonenumber](https://www.npmjs.com/package/google-libphonenumber). If you could suggest any other libraries to do the phone number validation, I can switch to that.

**Comment by CdTgr** [comment]
Here is a screenshot for reference.
<img width="1017" alt="image" src="https://user-images.githubusercontent.com/39024474/220988751-3ad06c9f-9569-49f6-9df9-67e4f8c906b1.png">

**Comment by andyslack** [comment]
I submitted a ticket on their repo - https://github.com/ruimarinho/google-libphonenumber/issues/361

Hopefully someone can take a look, I'm not great with tsc build/rollup, so not sure where to start on this, but if you have some free time and think you can fix it, feel free to submit a PR on their project.

**Comment by ruimarinho** [comment]
Hi,

I don't know if this part of code is actually evaluated for the purpose you're going to use it for (please double check this), but this comes directly from [Google Closure Library](https://github.com/google/closure-library/blob/9c4766f2edd40f8432a310ae4919494b737714f5/closure/goog/base.js#L1096), which is a direct dependency of libphonenumber. I don't think there's anything I can do to change this on my side. See similar concerns at https://github.com/google/libphonenumber/pull/2605#issuecomment-870905467.

**Comment by CdTgr** [comment]
@ruimarinho There is an alternative library (libphonenumber-js)[https://www.npmjs.com/package/libphonenumber-js] for the number formatting. I will change the code to use this library and so the issue would be solved.

**Comment by CdTgr** [comment]
the issue is resolved in [v1.1.0](https://www.npmjs.com/package/vue3-q-tel-input/v/1.1.0)
